src/app/create-event/page.tsx

```diff
@@ -27,12 +27,7 @@ type DefaultValues = {
 };
 
 const CreateEventPage: NextPage = () => {
-  const {
-    register,
-    watch,
-    handleSubmit,
-    formState: { errors },
-  } = useForm({
+  const methods = useForm<DefaultValues>({
```

Comment:
DefaultValues is very generic name, use something as specific for the use case as possible. In this case, there is no generic, this is `CreateEventValues` or `CreateEventFormInput` or `CreateEventInput` whatever you prefer